### DISCUSSION TITLE
Why was the original project forked?

### DISCUSSION THREAD

**Discussion by anton-suprun** [psyirius/Rockite#12] [Q&A] [answered]
Hi 👋 

Awesome tool! Really convenient for quickly testing / debugging WebSockets. 

I'm curious; why did you decide to  create a fork from the original project?

**Comment by psyirius** [comment, marked as answer]
Looks like the Original Project is Not maintained anymore and it doesn't seem like the author of the Original Project is active.

I had a personal private fork of it for almost an year with fixes that I needed. (like binary messages and such)
But, it came to my attention that there is no convenient tool that you can use to test a WebSocket endpoint quickly without login or signup.
And more and more tools these days love to hook their users up with an account which I don't see a good thing and it's privacy concerning in one way or another.

Thus, the fork :)

Glad, It's useful. Issues, feature request and PR's are welcome.
I hope to keep it up-to-date and make it available to multiple platforms as passible in my spare times.

**Reply by anton-suprun** [reply]
Ah I see. Thanks for the explanation. I didn't realize that the other project was no longer maintained!

Great work; thank you for doing this! 

> But, it came to my attention that there is no convenient tool that you can use to test a WebSocket endpoint quickly without login or signup.

Indeed! I also couldn't find anything else besides your tool which helped immensely in quickly testing my set-up. 

 > Glad, It's useful. Issues, feature request and PR's are welcome.


I'd be happy to contribute in the future:). I don't have capacity right now; working on another project right now. But sometime in the next year I'd be happy to contribute. Some of the work mentioned on the issues you made and the roadmap seem like very interesting problems to solve :) 

Thanks! 🙏
